cli_reference/admin_cli_operations.adoc

```diff
@@ -0,0 +1,171 @@
+= Administrator CLI Operations
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+This topic provides information on the administrator CLI operations and their syntax. You must link:get_started_cli.html[setup and login] with the CLI before you can perform these operations.
```

Comment:
Even if it's pretty obvious, we don't actually ever say anywhere plainly "the administrator CLI (is? / uses?) the oadm command" or similar. Seems like the Overview would be a good place to make that clear from the jump. Same is true of the basic_cli topic; it doesn't actually say "the OpenShift CLI (is? / uses?) the oc command", it just talks about "the CLI" then starts mentioning oc.

Again, it's fairly obvious in both topics, but it seems like a better intro would say so more clearly.